crates/core/src/db/models.rs

```diff
@@ -694,15 +694,24 @@ pub struct StoreCreator<'a> {
 }
 
 /// A row in the `graph_connections` table
-#[derive(Debug, Clone, Queryable, Insertable, AsChangeset)]
+#[derive(Debug, Clone, QueryableByName, Insertable, AsChangeset)]
```

Comment:
Any reason for the `QueryableByName` derive in this PR?

Reply:
No specific reason 😅